### ISSUE TITLE
Allow more people to collaborate directly in the repo

### ISSUE THREAD

**Issue by SpiritQuaddicted** [hrehfeld/QuakeInjector#122]
I see no issues with letting more people commit directly to the repo and maybe even prepare new releases. In fact, I see a need ;)

Let's add [anyone who made a PR in the past](https://github.com/hrehfeld/QuakeInjector/pulls?q=is%3Apr+is%3Aclosed) to the collaborators if they aren't already:
- @ericwa 
- @neogeographica 
- @Warsen 
- @KhevinLiepp
- @duganchen

**Comment by hrehfeld** [comment]
@ericwa @neogeographica already should have full access.
I added @khevinliepp and @duganchen , but couldn't associate @Warsen to the quake scene anymore.

**Comment by Warsen** [comment]
Although I did remove my fork of your repository after the successful pull request (this is best practice), I am willing to come back and make additional changes or bug fixes if I ever get my Java environment IDE set up again. Eclipse made me upset when they failed to give me a dependency I needed to compile last time. I'm fine with pull requests and not necessarily collaborator access.

May I suggest you compile and release an Alpha031 or Alpha04 release to help the person who created #111 ? There may not be much changes but every little bit helps.
